run-test.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+cd "$( dirname "${BASH_SOURCE[0]}" )"
+
+export VMCK_URL='http://10.66.60.1:8000'
+
+function clean {
+    vagrant destroy -f || echo "vagrant destroy failed, but we don't care"
+trap clean  EXIT
```

Comment:
Does this work if it's inside the function? I expect `trap` gets called only when `clean` is called, but maybe i'm wrong.

Reply:
Actually when the script exits it calls clean. So trap calls `clean`.

Reply:
> Actually when the script exits it calls clean

Does it? Howcome?

Reply:
That again is my bad. Now I see what you meant by `inside the function`. Nope! It should not work.... I'll fix it too. I don't know how it got inside the function